sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_models.py

```diff
@@ -411,18 +401,15 @@ def __init__(
         call_connection_id: Optional[str] = None,
         server_call_id: Optional[str] = None,
         targets: Optional[List[CommunicationIdentifier]] = None,
-        call_connection_state:
-        Optional[Union[str, 'CallConnectionState']] = None,
+        call_connection_state: Optional[Union[str, 'CallConnectionState']] = None,
         callback_url: Optional[str] = None,
         media_subscription_id: Optional[str] = None,
         source_caller_id_number: Optional[PhoneNumberIdentifier] = None,
         source_display_name: Optional[str] = None,
         source: Optional[CommunicationIdentifier] = None,
         correlation_id: Optional[str] = None,
-        answered_by: Optional[CommunicationUserIdentifier] = None,
-        **kwargs
```

Comment:
This model _is_ already GA - but is only ever used as a response, so a customer should never construct this - so I removed kwargs.

Reply:
This also applies to a few other models, like CallParticipant, anything "~Properties" and anything "~Result".